templates/gitlab.rb.j2

```diff
@@ -67,5 +73,7 @@ nginx['ssl_verify_client'] = "{{ gitlab_nginx_ssl_verify_client }}"
 nginx['ssl_client_certificate'] = "{{ gitlab_nginx_ssl_client_certificate }}"
 {% endif %}
 
+prometheus['listen_address'] = '{{ gitlab_prometheus_listen_address }}'
```

Comment:
It looks like this line snuck into your PR too; can you remove it from this PR so I can cleanly merge just the smtp bits?